uber/models.py

```diff
@@ -648,6 +648,9 @@ def change_badge(self, attendee, badge_type, badge_num=None):
         def valid_attendees(self):
             return self.query(Attendee).filter(Attendee.badge_status != c.INVALID_STATUS)
 
+        def all_attendees(self):
+            return self.staffers(only_staffing=False)
+
```

Comment:
This call does not pull "all attendees," it pulls "all attendees _signed up for shifts_." I understand the desire to create a sensible name for readability, but since I had to look up what `self.staffers(only_staffing=False)` did anyway, I don't think this particular name is super helpful.